Cargo.toml

```diff
@@ -144,6 +146,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_yaml = "0.9"
 sha3 = "0.10.6"
+smallvec = "1.10.0"
```

Comment:
We already depend on `tinyvec`, cannot we use that?

Reply:
The type stored in `TinyVec` must implement `Default`, It is not feasible to do that for Entities, if you insist on using this we could store `Option<Entity>`.

Reply:
Or we can create a wrapper type which implements Default (e.g. `Entity::PLACEHOLDER`).